src/components/Forms/DateField.js

```diff
@@ -1,15 +1,26 @@
 import React from 'react'
 import createAutoCorrectedDatePipe from 'text-mask-addons/dist/createAutoCorrectedDatePipe'
-import MaskedTextField from './MaskedTextField'
+import { omit } from 'underscore'
```

Comment:
I'm not sure if we use underscore, I don't think we do?

Reply:
Take it back, we still have it as a dependency but it's not being used anywhere anymore. If it's not too much of a pain can you do this without `omit`? Would help keep the size down.

Reply:
Added a simple `omit` function to the `utils` folder instead.